azure-iot-device/azure/iot/device/iothub/pipeline/pipeline_stages_iothub.py

```diff
@@ -108,7 +107,7 @@ def _handle_pipeline_event(self, event):
         if isinstance(event, pipeline_events_iothub.TwinDesiredPropertiesPatchEvent):
             # remember the $version when we get a patch.
             version = event.patch["$version"]
-            logger.info(
+            logger.debug(
```

Comment:
may be this should be info as well

Reply:
No, but I think the one below should be.

---
In reply to: [477604528](https://github.com/Azure/azure-iot-sdk-python/pull/649#discussion_r477604528) [](ancestors = 477604528)